be/etp-backend/src/main/resources/db/changelog/changelog-main.xml

```diff
@@ -13,12 +13,71 @@
       <column name="side" type="char(1)">
         <constraints nullable="false"/>
       </column>
-      <column name="limit_price" type="double">
+      <column name="price" type="double">
```

Comment:
This is technically wrong. We are never supposed to change previous Liquibase changesets. I should have done this in a new <changeSet> through an XML command for altering the name of the column. It's "okay" right now because we have not deployed this server anywhere. On a deployed server, this <changeSet id="1"> would not re-run, so the column name change would not happen.